sorbet/rbi/todo.rbi

```diff
@@ -4,6 +4,5 @@
 
 # typed: false
 
-module RDoc::Task; end
```

Comment:
I have another PR addressing this update https://github.com/Shopify/shopify_api/pull/980